tests/files/features/.lagoon.yml

```diff
@@ -35,6 +35,10 @@ environments:
       schedule: "H * * * *"
       command: printf '%s_%s' "CRONJOB" "H" >> /files/cron.txt
       service: node
+    - name: test removal
+      schedule: "* * * * *"
```

Comment:
this will not create a native cronjob object, but instead an inside-pod-cronjob which we know they work?
see https://github.com/amazeeio/lagoon/pull/1178/files#diff-f5443eafb5de7b143fbba445a79a5ee8R31 how to force a native cronjob

Reply:
actually let's test both: removal for native and inside-pod cronjobs

Reply:
Whoops; I got the native vs in pod time limits backwards. I'll fix that on the next update.